### PR TITLE
Update Adapter Docs

### PR DESCRIPTION
**Pull request by harishmohanraj**

### CODE REVIEW

File: docs/docs/en/user-guide/adapters/images/custom_fastapi_nats.png
Comment (davorinrusevljan) [inline docs/docs/en/user-guide/adapters/images/custom_fastapi_nats.png:1]: (e.g HTML/JS)

add. after g, and perhaps give some other examples like:
(e.g. HTML/JS, golang, java, ...)

File: docs/docs/en/user-guide/adapters/index.md [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
+When to Use the [**`FastAPIAdapter`**](../../api/fastagency/adapters/fastapi/FastAPIAdapter.md):
 
-- **Custom Client Applications**: If you want to build your **own client applications** that interact with your FastAgency workflows using REST APIs, this adapter provides greater flexibility and control over the client-side implementation.
-- **Moderate User Demand**: The FastAPI Adapter is a good fit for scenarios with **moderate user request volume**. For example, it's well-suited for a medium-sized company developing an internal custom chat application.
-- **Simplified Production Setup**: Choose this adapter if you need a **simple and easy-to-manage** production setup for deploying your FastAgency workflows as a REST API.
+- **Custom Client Applications**: If you want to build your **own client applications** in a different language, (e.g., [**HTML**](https://en.wikipedia.org/wiki/HTML){target="_blank"}/[**JavaScript**](https://en.wikipedia.org/wiki/JavaScript){target="_blank"}), that interacts with your FastAgency workflows using [**REST API**](https://en.wikipedia.org/wiki/REST){target="_blank"} and [**WebSockets**](https://en.wikipedia.org/wiki/WebSocket){target="_blank"}.
```
Comment (davorinrusevljan) [inline docs/docs/en/user-guide/adapters/index.md:22]: also add few more examples, like golang, java, C# ..

File: docs/docs/en/user-guide/adapters/images/custom_fastapi_nats.png
Comment (harishmohanraj) [reply docs/docs/en/user-guide/adapters/images/custom_fastapi_nats.png:1]: Added Golang in the image

File: docs/docs/en/user-guide/adapters/index.md (hunk 1 above)
Comment (harishmohanraj) [reply docs/docs/en/user-guide/adapters/index.md]: Added a few more examples.

**Review by davorinrusevljan — APPROVED** [review]

**Pull request merged**